src/reaper_osara.cpp

```diff
@@ -2424,6 +2424,20 @@ void cmdMoveToPrevItemKeepSel(Command* command) {
 	}
 }
 
+void cmdPropertiesFocus(Command* command) {
+	switch (fakeFocus) {
+		case FOCUS_ITEM:
+		case FOCUS_TAKEMARKER:
+			Main_OnCommand(40009, 0); // Item properties: Show media item/take properties
+			break;
+		case FOCUS_AUTOMATIONITEM:
```

Comment:
As you suspected, this should include FOCUS_ENVELOPE. The reason is that you might move to an automation item, then move to an envelope point within that item, which will set FOCUS_ENVELOPE... but the automation item is still selected.
```suggestion
		case FOCUS_AUTOMATIONITEM:
		case FOCUS_ENVELOPE:
```